src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -10,6 +10,7 @@
 import org.gwtbootstrap3.client.ui.base.TextBoxBase;
 import org.gwtbootstrap3.client.ui.constants.ValidationState;
 import org.gwtbootstrap3.client.ui.html.Div;
+import org.gwtbootstrap3.client.ui.html.Paragraph;

```

Comment:
Use a paragraph when rendering the Bio section.  https://sagebionetworks.jira.com/browse/SWC-5773